instance/tests/models/test_openedx_site_configuration_mixin.py

```diff
@@ -128,3 +128,109 @@ def test_html_elements_and_attributes_in_static_content_overrides(self):
             ]
         }
         self.assertEqual(yaml.safe_load(instance.get_site_configuration_settings()), expected_variables)
+
+    def test_app_server_site_configuration_override(self):
+        """
+        Test that when the static content overrides contain html elements and attributes, there are no errors and the
+        expected ansible SiteConfiguration variables are generated.
+        """
+        instance = OpenEdXInstanceFactory()
+        configuration_extra_settings = {
+            'EDXAPP_SITE_CONFIGURATION': [
+                {
+                    'values': {
+                        'override': True,
+                        'CONTACT_US_CUSTOM_LINK': '/random/place/'
+                    }
```

Comment:
Small nit: Trailing comma please.

```suggestion
                    },
```